COMMUNITY DATA: Include Community Feedback (#35)
numfocus/disc-unconference-2023-projects: unconference/methodologies_collecting_data.md

 - Compare year over the year NPS, create dashboard (pending - can we create a dashboard)

#### Community: Targeted Respondents

> CONTRIBUTORS: People who contribute to an OS project. Contributions can be code, non code, or both. Contributors can also be users but for the purposes of this survey we’re asking them to evaluate their experience as Contributors. 

Reviewer: ```suggestion
CONTRIBUTORS: People who contribute to an OS project. CONTRIBUTIONS ARE NOT LIMITED TO CODE. Contributors can also be users but for the purposes of this survey we’re asking them to evaluate their experience as Contributors.
```

 - Code contributor - a person who opens a pull request
 - Non-code contributor
   - Organizers/speakers for events, meetings, conferences
   - Writing, editing and updating documentation
   - UI or UX Design
   - Testing

>    - Operations activities that move forward the goals of a project, such as but not limited to: project management, social media administration, communications administration, fundraising, governance

Reviewer 2: I get kind of a duality feeling from reading this which I don't think should be there. At least to me (someone who is known to be a bit too "cuadriculado"), I perceive a first level/split: code vs non-code, then the non-code half has further subdivisions. This leaves me with the impression that code and non-code are equiparable, thus, all non-code contributions _together_ can be compared/comparable to code contributions. I also get the "contributions aren't only code" message and "all contributions are valuable" which imo deserve more emphasis.

An idea to try and emphasize more these last two messages could be to remove mentions to code/non-code and to have a single level bullet point list. The content would be basically the same:

```
CONTRIBUTORS: People who contribute to an OS project. CONTRIBUTIONS ARE NOT LIMITED TO CODE. Contributors can also be users but for the purposes of this survey we’re asking them to evaluate their experience as Contributors.
 - Code contributor - a person who opens a pull request
 - Organizers/speakers for events, meetings, conferences
 - Writing, editing and updating documentation
 - UI or UX Design
 - Testing
 - Operations activities that move forward the goals of a project, such as but not limited to: project management, social media administration, communications administration, fundraising, governance
```

Reviewer: Your suggestion makes sense, thank you for sharing it!

Author: Oriol - saludos, I hope all is well with you! I was away from Jupyter for a few months, apologies for the delay here. I think this is a great suggestion and feedback. Anavelyz and I worked on this today and have made some changes based on your comments. If you have time to take a look we have requested your review.
